conanfile.py

```diff
@@ -0,0 +1,31 @@
+from conans import ConanFile, CMake
+
+import os
+
+
+class Tentris_Parser(ConanFile):
+    name = "cppitertools"
+    version = "1.0"
+    author = "Ryan Haining"
+    description = "Implementation of python itertools and builtin iteration functions for C++17 "
+    topics = ("please add topics here")
+    settings = "build_type", "compiler", "os", "arch"
+    generators = "cmake", "cmake_find_package", "cmake_paths"
+
+    exports_sources = list()
```

Comment:
```
exports_sources = [f for f in os.listdir('.') if f.endswith('.hpp')]
```

the file name is already a string, list comprehension is shorter, `file` is a python builtin so you should not use it yourself.